src/yul-easycrypt/arguments.rs

```diff
@@ -0,0 +1,92 @@
+//!
+//! Yul to EasyCrypt translation
+//!
+
+use std::path::Path;
+use std::path::PathBuf;
+
+use path_slash::PathExt;
+use structopt::StructOpt;
+
+/// Describes the accepted command line arguments.
+#[derive(Debug, StructOpt)]
+#[structopt(name = "The EraVM Yul to CoreYul transpiler")]
+pub struct Arguments {
+    /// Print the version and exit.
+    #[structopt(long = "version")]
+    pub version: bool,
+
+    /// Input file name
+    pub inputs: Vec<String>,
+
+    /// Create one file per component and contract/file at the specified directory, if given.
+    #[structopt(short = "o", long = "output-dir")]
+    pub output_directory: Option<PathBuf>,
+
+    /// Enable system contract compilation mode.
+    /// In this mode EraVM extensions are enabled. For example, calls to addresses `0xFFFF` and below
+    /// are substituted by special EraVM instructions.
+    /// In the Yul mode, the `verbatim_*` instruction family is available.
+    #[structopt(long = "system-mode")]
```

Comment:
Let's turn it into `--enable-eravm-extensions`, like in [this PR](https://github.com/matter-labs/era-compiler-solidity/blob/e6007464b907ee0916b6e85fd062410bb0583a17/src/zksolc/arguments.rs#L157).